modules/exploits/linux/local/vmware_workspace_one_access_cve_2022_22960.rb

```diff
@@ -0,0 +1,145 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'builder'
```

Comment:
I don't think you use builder here?  This was needed here: https://github.com/rapid7/metasploit-framework/blob/40613a4741f099246cf932460894826900508afe/modules/exploits/linux/http/vmware_workspace_one_access_vmsa_2022_0011_chain.rb#L9